base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -249,9 +249,9 @@ impl ConsensusConstants {
         // setting sha3/monero to 40/60 split
         algos.insert(PowAlgorithm::Sha3, PowAlgorithmConstants {
             max_target_time: 180,
```

Comment:
```suggestion
            max_target_time: 1800,
```

Reply:
lol `// TODO: reset to previous value` - thanks